# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1359395856 -3600
#      Mon Jan 28 18:57:36 2013 +0100
# Node ID 200dab2eecd4980b085f8046b890118ea40ebf81
# Parent  131d40cd805b758632323cdfa0b1faf0315dfd53
gui: complete changeset 6c0fce0632a4 with #ifdef HAVE_QSCINTILLA for editor

* main_window.cc (construct,handle_editor_visible): added #ifdef HAVE_QSCINTILLA

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -451,18 +451,20 @@ main_window::handle_workspace_visible (b
   if (visible && !_workspace_view->isFloating ())
     focus_workspace ();
 }
 
 void
 main_window::handle_editor_visible (bool visible)
 {
   // if changed to visible and widget is not floating
+#ifdef HAVE_QSCINTILLA
   if (visible && !_file_editor->isFloating ())
     focus_editor ();
+#endif
 }
 
 void
 main_window::handle_documentation_visible (bool visible)
 {
   // if changed to visible and widget is not floating
   if (visible && !_documentation_dock_widget->isFloating ())
     focus_documentation ();
@@ -993,18 +995,20 @@ main_window::construct ()
   connect (_terminal_dock_widget,       SIGNAL (visibilityChanged (bool)),
            this,                        SLOT (handle_command_window_visible (bool)));
   connect (_workspace_view,             SIGNAL (visibilityChanged (bool)),
            this,                        SLOT (handle_workspace_visible (bool)));
   connect (_history_dock_widget,        SIGNAL (visibilityChanged (bool)),
            this,                        SLOT (handle_command_history_visible (bool)));
   connect (_files_dock_widget,          SIGNAL (visibilityChanged (bool)),
            this,                        SLOT (handle_current_directory_visible (bool)));
+#ifdef HAVE_QSCINTILLA
   connect (_file_editor,                SIGNAL (visibilityChanged (bool)),
            this,                        SLOT (handle_editor_visible (bool)));
+#endif
   connect (_documentation_dock_widget,  SIGNAL (visibilityChanged (bool)),
            this,                        SLOT (handle_documentation_visible (bool)));
 
   connect (reset_windows_action,        SIGNAL (triggered ()),
            this,                        SLOT   (reset_windows ()));
 #ifdef HAVE_QSCINTILLA
   connect (this,                        SIGNAL (settings_changed ()),
            _file_editor,                SLOT   (notice_settings ()));
